stubs/click-web/click_web/web_click_types.pyi

```diff
@@ -1,17 +1,19 @@
 import re
-import typing as t
+from typing import TypeVar
 
 import click
 
+_T = TypeVar("_T")
+
 class EmailParamType(click.ParamType):
     EMAIL_REGEX: re.Pattern[str]
-    def convert(self, value: t.Any, param: click.Parameter | None, ctx: click.Context | None) -> t.Any: ...
+    def convert(self, value: str, param: click.Parameter | None, ctx: click.Context | None) -> str: ...
 
 class PasswordParamType(click.ParamType):
-    def convert(self, value: t.Any, param: click.Parameter | None, ctx: click.Context | None) -> t.Any: ...
+    def convert(self, value: _T, param: click.Parameter | None, ctx: click.Context | None) -> _T: ...
 
 class TextAreaParamType(click.ParamType):
-    def convert(self, value: t.Any, param: click.Parameter | None, ctx: click.Context | None) -> t.Any: ...
+    def convert(self, value: _T, param: click.Parameter | None, ctx: click.Context | None) -> _T: ...
```

Comment:
@PyHedgehog since you added this stub in #13208
Was there a specific reason to have these as `Any`?